os/arch/arm/src/armv8-m/up_exception.S

```diff
@@ -420,4 +451,13 @@ g_nestlevel:
 	.size   g_nestlevel, 4
 #endif
 
+	.bss
+	.global	g_bakirqstkalloc
```

Comment:
This is not required for this PR, but how about printing the contents of this stack when asserted?
User stack and irq stack are printed from `up_assert`.

Reply:
Yes I need to add the new stack in few places like stackcheck, etc.. I will do this soon.